libs/sdk-ui-kit/src/Dialog/DialogCloseButton.tsx

```diff
@@ -0,0 +1,21 @@
+// (C) 2022-2025 GoodData Corporation
+import React from "react";
+import { IDialogCloseButtonProps } from "./typings.js";
+import { Button } from "../Button/index.js";
+
+export const DialogCloseButton = React.memo<IDialogCloseButtonProps>(function DialogCloseButton({
+    accessibilityConfig,
+    onCancel,
```

Comment:
to have both handlers on props interface make it quite unusual. Keep only one and let place where it is used to decide which callback to propagate